README.md

```diff
@@ -9,49 +9,34 @@
 
 [OpenVoiceOS](https://openvoiceos.org/) is an open source platform for smart speakers and other voice-centric devices.
 
-[Mycroft](https://mycroft.ai) is a hackable, open source voice assistant by MycroftAI. OVOS-core is a
-backwards-compatible descendant of [Mycroft-core](https://github.com/MycroftAI/mycroft-core), the central component of
-Mycroft. It contains extensions and features not present upstream. All Mycroft Skills and Plugins should work normally
-with OVOS-core. Other Mycroft-based assistants are also believed, but not guaranteed, to be compatible.
+[Mycroft](https://mycroft.ai) was a hackable, open source voice assistant by the now defunct MycroftAI. OpenVoiceOS continues that work and ovos-core (this repo) is the central component.
+
+All Mycroft Skills and Plugins should work normally with OVOS-core. Other Mycroft-based assistants are also believed, but not guaranteed, to be compatible.
 
 The biggest difference between OVOS-core and Mycroft-core is that OVOS-core is fully modular. Furthermore, common
 components have been repackaged as plugins. That means it isn't just a great assistant on its own, but also a pretty
 small library!
 
-Furthermore, it offers a number of cli bindings. The old Mycroft shell scripts still exist, and still work, but that
-stuff is now built into the Python program (docs to follow in the form of `--help`, because it's a lot.)
-
----
-
-**Installing OVOS-core** (NOTE: at this early stage, required system libs are presumed, and your distribution might be a
-question mark.)
-
-We suggest you do this in a virtualenv:
-
-`pip install ovos-core[mycroft]`
-
----
-
-As always, the OpenVoiceOS team thanks the following entities (in addition to MycroftAI) for making certain code and/or
-manpower resources available to us which may not have been compatible with our practices before:
-
-- NeonGecko
-- KDE
-- Blue Systems
-
 ## Table of Contents
 
-- [Running Mycroft](#running-mycroft)
+- [Installing OVOS](#installing-ovos)
 - [Skills](#skills)
 - [Getting Involved](#getting-involved)
 - [Links](#links)
 
+
+## Installing OVOS
+
+We strongly suggest you use the companion project [ovos-installer](https://github.com/OpenVoiceOS/ovos-installer) to install OVOS, several repositories are needed to get the full assistant running
+
 You can find detailed documentation over at the [community-docs](https://openvoiceos.github.io/community-docs) or [ovos-technical-manual](https://openvoiceos.github.io/ovos-technical-manual)
 
+This repo can be installed standalone via `pip install ovos-core`
```

Comment:
Not sure this line will do much other than confuse users who will think that means "I can install OVOS by typing this command..." It's probably good to mention this can be installed, but explain the circumstances where that makes sense (using as a "kernel" for your own assistant, dev environments, etc)

Reply:
There is a [section in the docs](https://openvoiceos.github.io/community-docs/042-install_ovos_core/), but it has not been updated to 0.0.8ax.